src/devices/bus/isa/svga_paradise.cpp

```diff
@@ -115,6 +129,20 @@ ROM_START( pvga1a_jk )
 	// TODO: consider splitting if a good dump surfaces
 	ROM_SYSTEM_BIOS(1, "nvga2", "Nokia NVGA2")
 	ROMX_LOAD( "nokia.vbi",    0x000000, 0x06000, BAD_DUMP CRC(9f430ae7) SHA1(3d37b86853347d43ebc85a7e92e4a609b13406bb), ROM_BIOS(1) )
+	// Please consider splitting, as it is used in: Olivetti M290, M300, M380XP and was a popular upgrade to the M24 (16bit card works in a 8bit slot)
+	// 16bit ISA graphics adapter - Chip: Paradise PVGA1A-JK - ROM: 2x27128, WD copyright - RAM: 256KB - OSC: 36MHz, 25.175MHz, 28.322 MHz - Connector: DB15 
+	// V1.06 ROMs have the labels "PDP5" and "PDP7"
+	ROM_SYSTEM_BIOS(2, "go481", "Olivetti GO481")
```

Comment:
This is not a real problem: an ISA client can override the default BIOS used, and the dump doesn't look more than an OEM to me. It's definitely not like Nokia NVGA2 where it clearly shows having more than default PVGA1.